deploy/template.yaml

```diff
@@ -742,6 +742,54 @@ Resources:
       FilterPattern: ""
       LogGroupName: !Ref IPVJourneyEngineFunctionLogGroup
 
+  IPVValidateCriCheckFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub "validate-cri-check-${Environment}"
+      Handler: uk.gov.di.ipv.core.validatecricheck.ValidateCriCheckHandler::handleRequest
+      Runtime: java11
+      PackageType: Zip
+      CodeUri: ../lambdas/validatecricheck
+      Architectures:
+        - x86_64
+      MemorySize: 512
+      Tracing: Active
+      Environment:
+        Variables:
+          ENVIRONMENT: !Ref Environment
+          POWERTOOLS_SERVICE_NAME: !Sub validate-cri-check-${Environment}
+          USER_ISSUED_CREDENTIALS_TABLE_NAME: !Ref UserIssuedCredentialsTable
+      Policies:
+        - DynamoDBReadPolicy:
+            TableName: !Ref UserIssuedCredentialsTable
+      Events:
+        IPVCoreInternalAPI:
```

Comment:
I think this has changed now. The internal gateway has changed to the private gateway. You might need to rebase off main to get those changes and see how this is now done

Reply:
Done